### PR TITLE
Add watchtower recipe

### PR DESCRIPTION
**Pull request by mcg1969**
This is a straightforward recipe, except that it cannot run the test suite, because it requires AWS credentials. Not sure what people thing about that...

### CODE REVIEW

File: recipes/watchtower/meta.yaml [hunk 1]
```diff
@@ -0,0 +1,49 @@
...
+    - python
+    - boto3 >=1.3.0
+
+test:
+  imports:
+    - watchtower
+
+about:
+  home: http://github.com/kislyuk/watchtower
+  license: Apache 2.0
```
Comment (jakirkham) [inline recipes/watchtower/meta.yaml]: Optionally can add `license_family: Apache`

**Comment by jakirkham** [comment]
> ...except that it cannot run the test suite, because it requires AWS credentials. Not sure what people thing about that...

If it is pure Python, we have not been so picky about test suites. The main motivation for test suites is to make sure that compiled code still works when we build it. Interpreted code doesn't tend to suffer from the same issues. Though having a few import tests can be handy.

**Comment by scopatz** [comment]
@mcg1969 any updates to this?

**Comment by mcg1969** [comment]
Sorry @scopatz, this sort of dropped off my radar because we stopped using it for a project. Let me see what I can do now.

**Comment by scopatz** [comment]
Thanks, just ping when you think it is ready

**Comment by mcg1969** [comment]
@kislyuk: this PR implements a conda-forge recipe for your `watchtower` package; it will allow conda packages to be built for it with minimal intervention (basically just a simple edit to the recipe whenever a new tagged version is released). Would you like to be added as an official recipe maintainer? If you do, you will receive notifications for activity that takes place with regards to this recipe.

**Comment by mcg1969** [comment]
@scopatz fixes made.

**Comment by scopatz** [comment]
Works on docker form me locally. Thanks!  Also, could you enable CircleCI for your fork of staged recipes, please? That way I don't have to test locally :)

**Comment by mcg1969** [comment]
@scopatz I'll do so! Thanks for the merges.

**Pull request merged**